doc/cephadm/services/monitoring.rst

```diff
@@ -184,10 +184,79 @@ To see the default container images, run a command of the following form:
+string representing the currently-running version of Ceph. In the Squid release
+of Ceph, this ``cephadm`` file is stored as a zip file and must be unzipped
+before its contents can be examined.
+
+This procedure explains how to generate a list of the default container images
+used by ``cephadm``.
+
+.. note:: This procedure applies only to the Sphinx release of Ceph. If you are
+   using a different version of Ceph, you cannot use this procedure to examine
+   the list of default containers used by cephadm. Make sure that you are
+   reading the documentation for the release of Ceph that you have installed.
+
+#. To create a directory called ``cephadm_dir``, run the following command:
+
+   .. prompt:: bash #
+
+      mkdir cephadm_dir
+
+#. To unzip ``/var/lib/ceph/{FSID}/cephadm.{DIGEST}`` in the directory
+   ``cephadm_dir``, run a command of the following form:
+
+   .. prompt:: bash #
+
+      unzip /var/lib/ceph/{FSID}/cephadm.{DIGEST} -d cephadm_dir > /dev/null
+
+   ::
+
+      warning [/var/lib/ceph/{FSID}/cephadm.{DIGEST}]:  14 extra bytes at
+      beginning or within zipfile
+      (attempting to process anyway)
+
+
+#. To use ``egrep`` to search for the string ``_IMAGE`` in the file
+   ``cephadm_dir_cephadmlib/constants.py``, run the following command: 
+
+   .. prompt:: bash #
+
+      egrep "_IMAGE" cephadm_dir/cephadmlib/constants.py
+
+   ::
+
+      DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:main'
+      DEFAULT_IMAGE_IS_MAIN = True
+      DEFAULT_IMAGE_RELEASE = 'squid'
+      DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.43.0'
+      DEFAULT_LOKI_IMAGE = 'docker.io/grafana/loki:2.4.0'
+      DEFAULT_PROMTAIL_IMAGE = 'docker.io/grafana/promtail:2.4.0'
+      DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.5.0'
+      DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.25.0'
+      DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/grafana:9.4.12'
+      DEFAULT_HAPROXY_IMAGE = 'quay.io/ceph/haproxy:2.3'
+      DEFAULT_KEEPALIVED_IMAGE = 'quay.io/ceph/keepalived:2.2.4'
+      DEFAULT_NVMEOF_IMAGE = 'quay.io/ceph/nvmeof:1.2.1'
+      DEFAULT_SNMP_GATEWAY_IMAGE = 'docker.io/maxwo/snmp-notifier:v1.2.1'
+      DEFAULT_ELASTICSEARCH_IMAGE = 'quay.io/omrizeneva/elasticsearch:6.8.23'
+      DEFAULT_JAEGER_COLLECTOR_IMAGE = 'quay.io/jaegertracing/jaeger-collector:1.29'
+      DEFAULT_JAEGER_AGENT_IMAGE = 'quay.io/jaegertracing/jaeger-agent:1.29'
+      DEFAULT_JAEGER_QUERY_IMAGE = 'quay.io/jaegertracing/jaeger-query:1.29'
+      DEFAULT_SMB_IMAGE = 'quay.io/samba.org/samba-server:devbuilds-centos-amd64'
+
 Default monitoring images are specified in
 ``/src/cephadm/cephadmlib/constants.py`` and in
 ``/src/pybind/mgr/cephadm/module.py``.
 
+*The information in the "Default Images" section was developed by Eugen Block
+in a thread on the [ceph-users] mailing list in April of 2024 and updated for
+Squid by Adam King in a thread on GitHub here:*
```

Comment:
```suggestion
Squid and Reef by Adam King in a thread on GitHub here:*
```
assuming we'll be updating the Reef one as well (which we should as it's also a zip)